assets/dev/js/editor/components/ai/hooks/use-prompt.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { setStatusFeedback } from '../api';
+
+const usePrompt = ( fetchData, config = {} ) => {
```

Comment:
It feels a little bit messy, I would maybe write this whole hook differently, let's talk about it if you want